Sources/Build/LLBuildManifestBuilder.swift

```diff
@@ -798,23 +830,28 @@ extension LLBuildManifestBuilder {
             )
         }
 
-        // Create a phony node to represent the entire target.
-        let targetName = target.target.getLLBuildTargetName(config: buildConfig)
-        let output: Node = .virtual(targetName)
+        if addTargetCmd {
+            self.addTargetCmd(
+                target: target.target,
+                isTestTarget: target.isTestTarget,
+                inputs: objectFileNodes
+            )
+        }
 
-        manifest.addNode(output, toTarget: targetName)
-        manifest.addPhonyCmd(
-            name: output.name,
-            inputs: objectFileNodes,
-            outputs: [output]
-        )
+        return objectFileNodes
+    }
+}
 
-        if plan.graph.isInRootPackages(target.target) {
-            if !target.isTestTarget {
-                addNode(output, toTarget: .main)
-            }
-            addNode(output, toTarget: .test)
-        }
+// MARK: - Compile Mixed Languages
+
+extension LLBuildManifestBuilder {
+    /// Create a llbuild target for a mixed target description.
+    private func createMixedCompileCommand(
+        _ target: MixedTargetBuildDescription
+    ) throws {
+        let clangOutputs = try createClangCompileCommand(target.clangTargetBuildDescription, addTargetCmd: false)
+        let swiftOutputs = try createSwiftCompileCommand(target.swiftTargetBuildDescription, addTargetCmd: false)
+        self.addTargetCmd(target: target.target, isTestTarget: target.isTestTarget, inputs: clangOutputs + swiftOutputs)
```

Comment:
seems weird that the values for the `inputs:` parameter have `Outputs` as a suffix

Reply:
This file is building a bipartite graph to represent the build. The output(s) of one node is passed as input(s) to another node.